src/SeleniumLibrary/keywords/browsermanagement.py

```diff
@@ -350,14 +350,45 @@ def switch_browser(self, index_or_alias):
         self.debug('Switched to browser with Selenium session id %s.'
                    % self.driver.session_id)
 
+    @keyword
+    def get_browser_ids(self):
+        """Returns index of all active browser as list.
+
+        See `Switch Browser` for more information and examples.
+
+        New in SeleniumLibrary 4.0
+        """
+        return self.drivers.active_driver_ids
+
+    @keyword
+    def get_browser_aliases(self):
```

Comment:
I can see this be useful for logging purposes. but does this keyword server some other purpose too?

Reply:
Logging and iterating over open browsers. 

Same as 'get id' this is also for be able to get what we set.

If some other tests opened a browser you can access with this keyword.